### PR TITLE
Feature 1713 adding widget seriously injured killed in bicycles scooter fix titles

### PR DESCRIPTION
**Pull request by tkalir**

### CODE REVIEW

**Comment by atalyaalon** [comment]
@ziv17 can you take a look?

File: anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py [hunk 1]
```diff
@@ -47,17 +45,16 @@ def get_seriously_injured_killed_in_bicycles_scooter(
...
+            else _("In") + " " + location_info["road_segment_name"]
 
     @staticmethod
     def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         subtitle = SeriouslyInjuredKilledInBicyclesScooterWidget.create_location_description(
-            request_params.location_info,
-            request_params.location_text)
-        items["data"]["text"] = {"title": TITLE,
+            request_params.location_info)
+        items["data"]["text"] = {"title": _("Number of severely injured or killed in bike, e-bike, or scooter accidents"),
```
Comment (ziv17) [inline anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py:57]: Hi,
The (unwritten ) agreement of the `Widget`s is that `items` is being generated in `generate_items()`, and localization is done in `localize_items()`. Future maintainers may not be aware that `create_location_description()` is part of the localization. I think it is best that the logic of creating the description will be performed during the `generate_items()`, and only the localization will be performed during the `localize_items()`.

Comment (tkalir) [reply anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py]: Hi, I moved the code from create_location_description to localize_items

I am not sure if you are saying that we should separate the description logic and the translation further, but if you do I think we should release this widget first and think about it later.

Comment (tkalir) [reply anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py]: Ok, I moved the code back to localize_items

**Review by ziv17 — APPROVED** [review]
Well done Tamar!

**Pull request merged**